frontend/src/metabase/public/containers/PublicOrEmbeddedDashboard/PublicOrEmbeddedDashboard.tsx

```diff
@@ -160,33 +143,6 @@ class PublicOrEmbeddedDashboardInner extends Component<PublicOrEmbeddedDashboard
     }
   }
 
-  getCurrentTabDashcards = () => {
```

Comment:
I could leave this in the controller, but I want the controller to only fetch data. This kind of logic should belong to view components.